lib/menu.rb

```diff
@@ -0,0 +1,26 @@
+class Menu
+  
+  attr_reader :menu
+  
+  def initialize
+    @menu = [
+      { :item => 1, :dish => "Mattar Paneer", :price => 12.50 },
+      { :item => 2, :dish => "Black Daal", :price => 7.50 },
+      { :item => 3, :dish => "Raita", :price => 3.50 },
+      { :item => 4, :dish => "Garlic Naan", :price => 3.50 }
+    ]
+  end
+  
+  def view 
+    @menu.map { |dish| "#{dish[:item]}. #{dish[:dish]} £#{dish[:price]}" }
```

Comment:
Were you to use just a single key/value hash as the one I suggested in my first comment, you could refactor this to something like: 

`puts "#{key}:#{value}"`

This will print the full menu.